src/index.js

```diff
@@ -7,133 +7,119 @@
 
 const express = require('express')
 
-const { init, connect } = require('@kiltprotocol/core')
+const { connect } = require('@kiltprotocol/core')
 const Did = require('@kiltprotocol/did')
 
+const {
+  W3C_DID_CONTEXT_URL,
+  KILT_DID_CONTEXT_URL
+} = require('@kiltprotocol/did')
 const { PORT, BLOCKCHAIN_NODE } = require('./config')
 const {
   URI_DID,
   DID_RESOLUTION_RESPONSE_MIME,
-  DID_DOC_CONTEXT,
-  KILT_DID_CONTEXT,
   DID_RESOLUTION_RESPONSE_CONTEXT
 } = require('./consts')
 
 const driver = express()
 
 async function start() {
-  await init({ address: BLOCKCHAIN_NODE })
-  const { api } = await connect()
-
-  const hasWeb3Names = () => !!api.consts.web3Names
-  const logWeb3NameSupport = () => {
-    console.info(
-      hasWeb3Names()
-        ? '🥳 Web3Names are available on this chain!'
-        : '👵 Web3Names are currently not available on this chain'
-    )
-  }
-  api.on('decorated', logWeb3NameSupport)
+  await connect(BLOCKCHAIN_NODE)
 
   // URI_DID is imposed by the universal-resolver
   driver.get(URI_DID, async (req, res) => {
     async function handleRequest(responseContentType) {
       const isJsonLd = responseContentType.includes('ld+json')
+      let didDocument = null
+      let didDocumentMetadata = {}
+      let didResolutionMetadata = {}
       // Catch-all for generic error 500
       try {
         console.log('--------------------')
         console.info('\n→ Received headers:')
         console.info(JSON.stringify(req.headers, null, 2))
         const { did } = req.params
 
-        const didResolutionResult = {
-          '@context': [DID_RESOLUTION_RESPONSE_CONTEXT],
-          didDocument: null,
-          didDocumentMetadata: {},
-          didResolutionMetadata: {
-            contentType: isJsonLd
-              ? 'application/did+ld+json'
-              : 'application/did+json'
-          }
+        // 1. resolve DID
+        ;({ didDocument, didDocumentMetadata, didResolutionMetadata } =
+          await Did.resolveCompliant(did))
```

Comment:
IS this a new function??

Reply:
it's 'new' in the sdk 0.31.0 or so. we just never updated the did driver